app/workers/evss/disability_compensation_form/workflow_steps.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module EVSS
+  module DisabilityCompensationForm
+    class WorkflowSteps
+      def set_has_uploads(id)
+        submission(id).update(has_uploads: true)
+      end
+
+      def set_has_form_4142(id)
+        submission(id).update(has_form_4142: true)
+      end
+
+      def uploads_success_handler(_status, options)
```

Comment:
@kreek : For batch failures, don't you need another method to update the column with false value? Likely set false along with a call to WorkflowSteps.new.set_has_uploads and then later on success it updates to true?

Reply:
The default value for the success columns is false :)